internal/pkg/skuba/upgrade/addon/versions.go

```diff
@@ -55,20 +56,35 @@ func UpdatedAddons(client clientset.Interface, clusterVersion *version.Version)
 	return aviu, nil
 }
 
+func addonsByName(addons kubernetes.AddonsVersion) []kubernetes.Addon {
+	sortedAddons := make([]kubernetes.Addon, len(addons))
+	i := 0
+	for addon := range addons {
+		sortedAddons[i] = addon
+		i++
+	}
+	sort.Slice(sortedAddons, func(i, j int) bool {
+		return string(sortedAddons[i]) < string(sortedAddons[j])
+	})
+	return sortedAddons
+}
+
 func PrintAddonUpdates(updatedAddons AddonVersionInfoUpdate) {
-	for addonName, versions := range updatedAddons.Updated {
+	for _, addonName := range addonsByName(updatedAddons.Updated) {
```

Comment:
```suggestion
	for _, addon := range addonsByName(updatedAddons.Updated) {
```

So it's clear that it's the addon itself, not the name (would hint it's a string)

Reply:
Thanks, updated!